.spelling

```diff
@@ -892,6 +892,9 @@ Lorem
 # To allow Site Init
 Init
 
+- operations/boot_orchestration/BOS_export_import.md
+Redis
```

Comment:
I'm doing a sweep to reorganize the `.spelling` file, application names will be moved to the global list.

Do you think within this PR that "Redis" could just be moved to the global list?

Reply:
Done.